packages/amplication-server/src/core/project/project.service.ts

```diff
@@ -65,6 +65,17 @@ export class ProjectService {
     args: ProjectCreateArgs,
     userId: string
   ): Promise<Project> {
+    const projectEntitlement = await this.billingService.getMeteredEntitlement(
+      args.data.workspace.connect.id,
+      BillingFeature.Projects
+    );
+
+    if (projectEntitlement && !projectEntitlement.hasAccess) {
+      throw new Error(
```

Comment:
Same here, we should keep using the BillingLimitationError for any exception related to limitation errors